Modules/timemodule.c

```diff
@@ -962,11 +939,11 @@ Monotonic clock, cannot go backward.");
 static PyObject*
 perf_counter(_Py_clock_info_t *info)
 {
-#ifdef WIN32_PERF_COUNTER
-    return win_perf_counter(info);
-#else
-    return pymonotonic(info);
-#endif
+    _PyTime_t t;
```

Comment:
Idea for avoiding the redundant `double` -> `_PyTime_t` -> `double` conversion in `time.perf_counter()`:

What if this still kept a compile time conditional branch between the Windows API and the \*nix one?